### ISSUE TITLE
issue resuming unfinished model training

### ISSUE THREAD

**Issue by helizabeth1103** [google/deepvariant#1042]
Hello again,

I have been attempting to continue model training runs in the way suggested in the [previous issue here](https://github.com/google/deepvariant/issues/1016), but they don't seem to be resuming from a checkpoint, rather starting over again every time. I am not changing anything in the script, so the --experiment-dir path does not change. Is there something additional I need to add to the script to indicate that the model should start from a previous checkpoint? If submitting the command as I have repeatedly was going to work to resume from a checkpoint, the model training should have finished by now. 

Attached are three consecutive log files from attempts to resume the model training from where it left off. Each starts at epoch 0, gets up to about 45-50% complete in epoch 4 and then times out.

[deepvariant_modeltrain-18828154-atlas-0244.err.txt](https://github.com/user-attachments/files/23952187/deepvariant_modeltrain-18828154-atlas-0244.err.txt)
[deepvariant_modeltrain-18819574-atlas-0244.err.txt](https://github.com/user-attachments/files/23952185/deepvariant_modeltrain-18819574-atlas-0244.err.txt)
[deepvariant_modeltrain-18822079-atlas-0245.err.txt](https://github.com/user-attachments/files/23952186/deepvariant_modeltrain-18822079-atlas-0245.err.txt)

Here is the script I am submitting successively:

apptainer exec --bind /90daydata,$TMPDIR --nv /90daydata/pbarc/haley.arnold/AI_ModelTraining/Samples/deepvariant_1.6.0.sif train
--config="/90daydata/pbarc/haley.arnold/AI_ModelTraining/Samples/dv_config.py:base"
--config.train_dataset_pbtxt="shuffledFM2s_chr5.with_label_channelsize.shuffled.pbtxt"
--config.tune_dataset_pbtxt="shuffledFM1s_chr5.with_label_channelsize.shuffled.pbtxt"
--config.init_checkpoint=gs://deepvariant/models/DeepVariant/1.6.1/checkpoints/wgs/deepvariant.wgs.ckpt
--config.num_epochs=10
--config.learning_rate=0.02
--config.num_validation_examples=0
--experiment_dir="model_chr5trainandval_newertruth_out/"
--strategy=mirrored
--config.batch_size=32

**Comment by akolesnikov** [comment]
Closing due to inactivity.
